services/director-v2/src/simcore_service_director_v2/modules/dask_client.py

```diff
@@ -310,7 +311,7 @@ def _comp_sidecar_fct(
                     input_data=input_data,
                     output_data_keys=output_data_keys,
                     log_file_url=log_file_url,
-                    command=["run"],
+                    command=node_image.command,
```

Comment:
you can also add in the  dask-sidecar code, that it prints the command in the logs as well. could be helpful for debugging

Reply:
You can find that some lines below ...

Reply:
indeed I did not see it modified, that should be in there:
https://github.com/sanderegg/osparc-simcore/blob/master/services/dask-sidecar/src/simcore_service_dask_sidecar/tasks.py#L1

Reply:
in place